### PR TITLE
[ETL-107] Add SQS template and config

### PR DESCRIPTION
**Pull request by philerooski**
This is just the SQS CFN template and an example config. The other work we've been doing (including moving the lambda to its own stack) will be included in the PR for ETL-117.

### CODE REVIEW

File: config/develop/sqs-queue.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+template_path: sqs-queue.yaml
+stack_name: sqsqueue
+parameters:
+  PrimaryQueueName: my_queue
```
Comment (tthyer) [inline config/develop/sqs-queue.yaml:4]: Let's give this a more descriptive name

File: config/develop/sqs-queue.yaml [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+  DeadLetterQueueName: my_dead_letter_queue
```
Comment (tthyer) [inline config/develop/sqs-queue.yaml:5]: and another more descriptive name

**Review by tthyer — CHANGES_REQUESTED** [review]
Looks good, I just need a few more descriptive names.

What did you do with the lambda-specific commits to the etl-107 branch that we were making?

**Comment by philerooski** [comment]
@tthyer I namespaced the SQS queue stack and borrowed the existing naming convention we've used for namespaced resources. I also added some SNS resources and policies to the SQS template in anticipation of removing those same resources from the lambda template.

File: templates/sqs-queue.yaml [hunk 3]
```diff
@@ -4,24 +4,14 @@ Description: >
...
-
 Resources:
 
   PrimaryQueue:
     Type: AWS::SQS::Queue
     Properties:
       DelaySeconds: 0
       MessageRetentionPeriod: 86400
-      QueueName: !Ref PrimaryQueueName
+      QueueName: !Sub '${AWS::StackName}-Queue'
```
Comment (tthyer) [inline templates/sqs-queue.yaml:14]: better

**Review by tthyer — APPROVED** [review]

**Pull request merged**